src/profile.js

```diff
@@ -126,7 +126,7 @@ module.exports = React.createClass({
     }
   },
   renderCurrentSeries: function(series){
-    if (series && series.completed === false){
+    if (series && series.active === true){
```

Comment:
Refactor suggestion line 129:
if (series.active) {

If there is a series, it will return true or false whether it's active or not, and if there is not series, it will return null, which also returns false.